core/wp-api.php

```diff
@@ -0,0 +1,39 @@
+<?php
+namespace Elementor\Core;
+
+use Elementor\Core\Utils\Collection;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * This class responsible for the interaction with WordPress Core API,
+ * the main benefit of this class is that it easy to mock in testing
+ * and it can help to create unit tests without the hustle of mocking WordPress itself.
```

Comment:
```suggestion
 * This class is responsible for the interaction with WordPress Core API.
 * The main benefit is making it easy to mock in testing
 * and it can help to create unit tests without the hustle of mocking WordPress itself.
```